pkg/querier/distributor_queryable_streaming_test.go

```diff
@@ -56,7 +56,7 @@ func TestStreamingChunkSeries_HappyPath(t *testing.T) {
 	require.Equal(t, 1.0, m.SumCounters("cortex_distributor_query_ingester_chunks_deduped_total"))
 
 	require.Equal(t, uint64(3), queryStats.FetchedChunksCount)
-	require.Equal(t, uint64(114), queryStats.FetchedChunkBytes)
+	require.Equal(t, uint64(111), queryStats.FetchedChunkBytes)
```

Comment:
Note to reviewers: I don't know why this changed, and I haven't had time to investigate it.

Reply:
Looks like this is due to a bug fix upstream: https://github.com/prometheus/prometheus/commit/d90d0976b54879e833ec08b8855603e8f4e4b432